Sources/RunnerLib/Runtime.swift

```diff
@@ -2,7 +2,7 @@ import Foundation
 
 //  Bunch of Danger runtime util funcs
 
-public struct Runtime {
+public enum Runtime {
     public static let supportedPaths = ["Dangerfile.swift", "Danger.swift", "danger/Dangerfile.swift", "Danger/Dangerfile.swift"]
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 12...
  
  
  DangerID: danger-id-Danger;
  File: Sources/RunnerLib/Runtime.swift;
  Line: 6;
-->

- :warning: Line should be 120 characters or less: currently 129 characters (`line_length`)